webapp-django/crashstats/crashstats/models.py

```diff
@@ -987,11 +987,19 @@ def get(self, bugs):
             url = settings.BZAPI_BASE_URL + (
                 '/bug?id=%(bugs)s&include_fields=%(fields)s' % params
             )
-            response = requests_retry_session().get(
+            response = requests_retry_session(
+                # BZAPI isn't super reliable, so be extra patient
+                retries=5,
+                # 502 = Bad Gateway
+                # 504 = Gateway Time-out
+                status_forcelist=(500, 502, 504),
```

Comment:
It might be more flexible to pass in a predicate here. Maybe call it `should_retry`. Then this could be like:

```
response = requests_retry_session(
    retries=5,
    should_retry=lambda resp: resp.status_code != 200,
)
```

or more like yours:

```
response = requests_retry_session(
    retries=5,
    should_retry=lambda resp: resp.status_code in (500, 502, 504)
)
```

I've seen that kind of API with network retry decorators.